pkg/collector/corechecks/snmp/snmp.go

```diff
@@ -161,30 +60,27 @@ func (c *Check) Configure(rawInstance integration.Data, rawInitConfig integratio
 		return fmt.Errorf("common configure failed: %s", err)
 	}
 
-	config, err := buildConfig(rawInstance, rawInitConfig)
+	c.config, err = checkconfig.NewCheckConfig(rawInstance, rawInitConfig)
 	if err != nil {
 		return fmt.Errorf("build config failed: %s", err)
 	}
+	log.Debugf("SNMP configuration: %s", c.config.ToString())
```

Comment:
This output will have the wrong IPAddress

Reply:
Is it the same topic as https://github.com/DataDog/datadog-agent/pull/8971#discussion_r706294917

I'm not sure to understand the comment or the issue to be addressed.

Reply:
You are calling `c.config.ToString()` while you assign the target IP address at line 69. Oh wait. Adding a comment at line 69